### PR TITLE
Introduced healthcare openmojis

### PR DESCRIPTION
**Pull request by b-g**
Hi @carlinmack, this PR introduces 44 healthcare related openmojis ... could you do me a favor and double check the names + add more tags in `data/extras-openmoji.csv`?

### CODE REVIEW

File: src/extras-openmoji/healthcare/E311.svg [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <path d="M37.4951,32.326c.0508-.2238.0889-.38.1215-.5365a11.8359,11.8359,0,0,1,1.2788-3.6313,3.2285,3.2285,0,0,1,2.6531-1.8412,4.3263,4.3263,0,0,1,2.22.5526,14.5931,14.5931,0,0,1,4.2462,3.5118,29.3667,29.3667,0,0,1,5.1242,8.2023,16.2156,16.2156,0,0,1,1.4317,6.6242c-.0117.5285.0393,1.0584.0614,1.5876v4.8619c-.0255.1974-.0414.3965-.0782.5918a23.7765,23.7765,0,0,1-.532,2.6653A2.9842,2.9842,0,0,1,51.7,57.0763a6.52,6.52,0,0,1-3.1888-.1633,17.3332,17.3332,0,0,1-10.1151-7.2761,8.9134,8.9134,0,0,1-1.5508-5.1524c.0494-2.6436.119-5.2869.1755-7.93A23.711,23.711,0,0,1,37.4951,32.326Z" fill="#92d3f5"/>
```
Comment (carlinmack) [inline src/extras-openmoji/healthcare/E311.svg]: why are the lungs blue? lungs are definitely red

Comment (b-g) [reply src/extras-openmoji/healthcare/E311.svg]: You are right, that's odd. Changed to red.

File: src/extras-openmoji/healthcare/E314.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/healthcare/E314.svg]: This emoji is currently listed as "appendix" but this is a large intestine emoji, can I change its name? Also I don't think the appendix translates well to an emoji as it's not very distinctive.

File: src/extras-openmoji/healthcare/E306.svg [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/healthcare/E306.svg]: This emoji is currently listed as 'patch', but this is a bandage, bandaid or plaster. A patch is closer to an eye patch that a pirate wears or something. I'm changing it's name to bandage.

File: src/extras-openmoji/healthcare/E312.svg [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/healthcare/E312.svg]: I think there is a glitch with the vertical line over the middle

File: src/extras-openmoji/healthcare/E313.svg [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/healthcare/E313.svg]: The fill should go to the very top and bottom of the strokes

File: src/extras-openmoji/healthcare/E310.svg [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/healthcare/E310.svg]: should be centered

File: src/extras-openmoji/healthcare/E309.svg [hunk 7]
```diff
@@ -0,0 +1,23 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/healthcare/E309.svg]: should be centered

**Comment by carlinmack** [comment]
no!!! how can I unmerge

**Comment by carlinmack** [comment]
the correct master, with no healthcare emojis, is in `master-test`. I'm scared to force-push to `master` to remove those commits as I'm not 100% sure what the behaviour is. 

The reason this happened is that I did some work then `git pull`ed, saw that the was no updates so thought I was good to push without realising I still had all the healthcare stuff in my master branch 🤦‍♂️

File: src/extras-openmoji/healthcare/E323.svg [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (b-g) [reply src/extras-openmoji/healthcare/E323.svg]: Fine for me ...

File: src/extras-openmoji/healthcare/E32A.svg [hunk 9]
```diff
@@ -0,0 +1,24 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (b-g) [reply src/extras-openmoji/healthcare/E32A.svg]: Removed ... not good enough.

**Comment by b-g** [comment]
@carlinmack Many thanks once more for the nice and helpful feedback! Addressed nearly everything in https://github.com/hfg-gmuend/openmoji/commit/70095bc21a601de4522a22565ada3370e2df1287. 🙏!

**Pull request merged**